storage/src/main/java/tech/pegasys/teku/storage/server/rocksdb/RocksDbHelper.java

```diff
@@ -0,0 +1,339 @@
+            totalSstFilesSizeLong,
+            totalBlobFilesSizeLong);
+      } catch (NumberFormatException e) {
+        LOG.error("Failed to parse string into long: " + e.getMessage());
+      }
+    }
+    // return empty usage on error
+    return new ColumnFamilyUsage(getNameById(cfHandle.getName(), spec), 0, 0, 0, 0);
+  }
+
+  public static void printTotals(
+      final SubCommandLogger out, final List<ColumnFamilyUsage> columnFamilyUsages) {
+    final long totalKeys = columnFamilyUsages.stream().mapToLong(ColumnFamilyUsage::keys).sum();
+    final long totalSize =
+        columnFamilyUsages.stream().mapToLong(ColumnFamilyUsage::totalSize).sum();
+    final long totalSsts =
+        columnFamilyUsages.stream().mapToLong(ColumnFamilyUsage::sstFilesSize).sum();
+    final long totalBlobs =
+        columnFamilyUsages.stream().mapToLong(ColumnFamilyUsage::blobFilesSize).sum();
+    printSeparator(out);
+    printLine(
+        out,
+        "ESTIMATED TOTAL",
+        String.valueOf(totalKeys),
+        formatOutputSize(totalSize),
+        formatOutputSize(totalSsts),
+        formatOutputSize(totalBlobs));
+    printSeparator(out);
+  }
+
+  private static boolean isPopulatedColumnFamily(final long size, final long numberOfKeys) {
+    return size != 0 || numberOfKeys != 0;
+  }
+
+  static String formatOutputSize(final long size) {
+    if (size > (1024 * 1024 * 1024)) {
+      long sizeInGiB = size / (1024 * 1024 * 1024);
+      return sizeInGiB + " GiB";
+    } else if (size > (1024 * 1024)) {
+      long sizeInMiB = size / (1024 * 1024);
+      return sizeInMiB + " MiB";
+    } else if (size > 1024) {
+      long sizeInKiB = size / 1024;
+      return sizeInKiB + " KiB";
+    } else {
+      return size + " B";
+    }
+  }
+
+  private static String getNameById(final byte[] id, final Spec spec) {
+    Map<String, KvStoreColumn<?, ?>> columnMap =
+        V6SchemaCombinedSnapshot.createV6(spec).getColumnMap();
+    for (Map.Entry<String, KvStoreColumn<?, ?>> entry : columnMap.entrySet()) {
+      if (Arrays.areEqual(entry.getValue().getId().toArray(), id)) {
+        return entry.getKey();
+      }
+    }
+    return null; // id not found
+  }
+
+  public static void printTableHeader(final SubCommandLogger out) {
+    printSeparator(out);
+    out.display(
+        "| Column Family                  | Keys            | Total Size  | SST Files Size  | Blob Files Size  | \n");
+    printSeparator(out);
+  }
+
+  private static void printSeparator(final SubCommandLogger out) {
+    out.display(
+        "|--------------------------------|-----------------|-------------|-----------------|------------------|\n");
+  }
+
+  static void printLine(
+      final SubCommandLogger out,
+      final String cfName,
+      final String keys,
+      final String totalFilesSize,
+      final String sstFilesSize,
+      final String blobFilesSize) {
+    final String format = "| %-30s | %-15s | %-11s | %-15s | %-16s |\n";
```

Comment:
we should fix these widths for our long column name, i think if we at least have `%-35s` we'll fit column names